com.remarkable.reMarkable.json

```diff
@@ -0,0 +1,70 @@
+{
+    "app-id": "com.remarkable.reMarkable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "reMarkable",
+    "finish-args": [
+        "--filesystem=home",
+        "--socket=x11",
+        "--share=network",
+        "--extra-data=remarkable-linux-client.tgz:1a1519adc6d62359f6f73bc2a4a6e76fb710fed78ca1bc09e46191e825d3058c:20777403::https://remarkable.engineering/remarkable-linux-client-0.0.5-16-1408-g7eca2b66.tgz",
```

Comment:
I assume this is an x86-64 only binary? You'll need a flathub.json file as well to limit it to that architecture as documented on the [maintainers page](https://github.com/flathub/flathub/wiki/Maintaining-an-application-on-flathub).

Reply:
Fixed